facts/get_facts.sh

```diff
@@ -2,6 +2,9 @@
 
 export PATH=/opt/puppetlabs/bin:$PATH
 
+# ensure IPv6 is always enabled, some boxes disable it by default, e.g. Fedora and RedHat
```

Comment:
I wonder why we even use `generic/fedoraNN` instead of `fedora/NN-cloud-base`. AFAIK the latter doesn't disable IPv6.